packages/bpk-component-slider/src/BpkSlider.tsx

```diff
@@ -16,60 +17,74 @@
  * limitations under the License.
  */
 
-import PropTypes from 'prop-types';
-
-// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
-import Slider from 'react-slider';
+import * as Slider from '@radix-ui/react-slider';
 
 import { cssModules, isRTL } from '../../bpk-react-utils';
 
 import STYLES from './BpkSlider.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-type Props = {
-  className?: string | null;
+export type Props = {
+  max: number
+  min: number
+  minDistance?: number,
+  step?: number
+  onChange: (value: number[] | number) => void
+  onAfterChange?: (value: number[] | number ) => void
+  value: number[] | number
+  ariaLabel: string[]
+  ariaValuetext?: string[]
   [rest: string]: any;
 }
 
-const BpkSlider = (props: Props) => {
-  const { className, ...rest } = props;
+const BpkSlider = ({ ariaLabel, ariaValuetext, max, min, minDistance, onAfterChange, onChange, step, value, ...rest }: Props) => {
   const invert = isRTL();
-  const classNames = [getClassName('bpk-slider')];
-  const thumbClassNames = [getClassName('bpk-slider__handle')];
-  const trackClassNames = [getClassName('bpk-slider__bar')];
+  const defaultValue = Array.isArray(value) ? value : [value]
+
+  const processSliderValues = (sliderValues: number[], callback?: (val: number | number[]) => void) => {
+  const val = sliderValues.length === 1 ? sliderValues[0] : sliderValues;
+  if (callback) {
+    callback(val);
+   }
+  };
+
+  const handleOnChange = (sliderValues: number[]) => {
+    processSliderValues(sliderValues, onChange);
+  };
 
-  const isRange = (rest.value || rest.defaultValue || []).length > 1;
+  const handleOnAfterChange = (sliderValues: number[]) => {
+    processSliderValues(sliderValues, onAfterChange);
+  };
 
-  if (isRange) {
-    classNames.push(getClassName('bpk-slider--range'));
-  }
-  if (className) {
-    classNames.push(getClassName(className));
-  }
 
   return (
-    <Slider
+    <Slider.Root
+      className={getClassName('bpk-slider')}
```

Comment:
Why is it that we remove className throughout, when we still need it here for the root styling?

Reply:
We still need to give the slider skyscanner the styles but removing the className prop from BpkSlider prevents the likes of Banana adding their own styles on top of that